version.go

```diff
@@ -0,0 +1,34 @@
+package hcsshim
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/Microsoft/hcsshim/internal/hcs"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+type SupportedSchemaVersionsStruct struct {

```

Comment:
>type [](http://example.com/codeflow?start=0&length=4)

suggest name this file service.go, so we can add service related functions to this file in the future

Reply:
fix